ProjectEureka/project-eureka-flutter
# PE-67 | Implemented a Side Menu Service

**What I Did:*
- Used the UserById Service created by @davidsquines to implement dynamic data into the side menu.
- Removed getAllUsers Service, since that is unneeded.

**How to Test:**
1. Sign in using any account.
2. Open the Side Menu and the account's name and email should show up.
3. Check all pages of the Side Menu.
